styles/style.css

```diff
@@ -361,3 +358,491 @@ input {
   text-align: center;
   margin-bottom: 5px;
 }
+
+/* Media Queries section for responsive layout mobile version */
+
+@media only screen and (max-width: 600px) {
+  .nav-menu {
+    display: none;
+  }
+
+  .hamburger {
+    display: flex;
+  }
+
+  .grid-container {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
- [ ] **OPTIONAL**
Please keep in mind that it doesn't make sense to set up two media screens with min and max of the same width of 600 px!
This will cause that both changes will only apply when the screen is on 600px, not 599px or 601px!
I think you should think a bit about how the media queries work to make the website fully responsive
**HINT**
`@media only screen and (min-width: 600px) and (max-width: 768px) // you can modify that of course, but the point is to define a width scale on the screen for the elements to change correctly.

Reply:
will work on this tomorrow.